src/modules/addMoreStats.js

```diff
@@ -1767,10 +1767,17 @@ function addMoreStats(){
 	}//end boneless check
 			return
 		};
-		if(user === whoAmI && reliablePersistentStorage){
-			cache.getList("ANIME",function(data){
-				personalStatsCallback(data)
-			})
+		if(user === whoAmI){
```

Comment:
Does APIv2 have its own equivalent of ``reliablePersistentStorage``?

Reply:
well it also uses localforage for cache which falls back to localStorage automatically if IndexedDb/persistent storage is unavailable